### PR TITLE
Fixed All Asked Changes

### PR DESCRIPTION
**Pull request by palak-chaturvedi**
1. PurpleMaze is not clickable
2. Remove doodle
3. Remove Resources
4. Pricing not working 
5. Login page
6. Profile not working
7. Mobile view aligned
8. Upgrade plan (Prices are in USD- INR)
9. Reset Password page
10. Forget Password page
11. Popups (SignUp Popup for verify email)

Done All

### CODE REVIEW

**Comment by himanshush200599** [comment]
@palak-chaturvedi are the changes of GRID that weren't working on Chrome are also fixed now?

And can you please add some screenshots as well for different screen size if possible?

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -14,88 +14,162 @@ export const Footer = ({
...
-            <div className="group">
-              <div className="img-wrapper">
-                <img className="img" alt="Img" src={img} />
+
+        <div class="flex flex-col lg:flex-row px-8 py-10">
+          <div class="flex flex-col lg:w-1/3 px-2">
+            <a href="/">
+              <div class="flex items-center">
+
+                <img className="px-2" alt="Img" src="" />
```
Comment (himanshush200599) [inline src/components/Footer/Footer.jsx]: [Doubt]
Why we are having here empty `img` tag?

Comment (palak-chaturvedi) [reply src/components/Footer/Footer.jsx]: I forgot about it. I have added the img tag now.
And yes the grid is working. Its not working in my device but Gagan checked its working.

Comment (palak-chaturvedi) [reply src/components/Footer/Footer.jsx]: @himanshush200599 Do you want screenshots of home page or everyscreen?

Comment (himanshush200599) [reply src/components/Footer/Footer.jsx]: The screenshot of UI's Screen for different screen-type  less than 500px, 760px, 1080px and one bigger 
@palak-chaturvedi

**Pull request closed without merging**